README.md

```diff
@@ -211,6 +211,14 @@ const text = await response.text();
 ...
 ```
 
+Fetch data from a workbook view in JSON format.
```

Comment:
We do not need to showcase every combination of method / options / value in these examples. For getting all the details, the apidoc is the better tool. That's what I tried to explain in the orange box here: https://conf.disy.net/pages/viewpage.action?spaceKey=EHB&title=Cadenza+JS#CadenzaJS-ExamplesintheREADME

So for me, another media type is not important enough to "justify" another two examples. What do you think?

Reply:
Agreed, one media type showcased with a code snippet is enough.